### ISSUE TITLE
Implement channel backups and restore

### ISSUE THREAD

**Issue by nolim1t** [lncm/noma#29]
Implement a function called:

noma lnd backup
noma lnd restore

**Comment by AnotherDroog** [comment]
I’ve got a “cloud” in the works so we don’t have to worry about that here. 

Backup and restore functions: +1

**Comment by nolim1t** [comment]
I have a proposal to include into the "autobackup" function. Also I'll be using the toml library too as invoicer uses the same config format.

Working proof of concept
https://gitlab.com/lncm/pi-factory/snippets/1860150

**Comment by AnotherDroog** [comment]
Taking a look

**Comment by AnotherDroog** [comment]
👍 Your proof of concept is good relying on inotity. 

While merging it would be nice to add scp over ssh.

**Comment by AnotherDroog** [comment]
Also see @meeDamian 's https://gist.github.com/meeDamian/36831e6d433031363dd2bf8561764a10

**Comment by meeDamian** [comment]
> Also see @meeDamian 's [gist.github.com/meeDamian/36831e6d433031363dd2bf8561764a10](https://gist.github.com/meeDamian/36831e6d433031363dd2bf8561764a10)

Note that this is very different.  It's intended to be run from an external device, there's zero trigger logic, as it relies on either scheduling it externally, or running it manually.

BTW: Has anyone tried to restore `lnd` from `zeed + static backup file`?

**Comment by AnotherDroog** [comment]
I also like how Arvinda is using asymmetric keys here https://github.com/vindard/lnd-backup

**Comment by nolim1t** [comment]
Looks cool but I don't think you need to really encrypt it as it complicates the restore process, and the static backup is directly tied to the node

**Comment by AnotherDroog** [comment]
That’s what I thought. The relevant files are only useful with the wallet passphrase

It would be nice to have certainty on this.

**Comment by AnotherDroog** [comment]
I've checked the lnd code, `channel.backup` is indeed encrypted with the seed's passphrase.

We can close this once #54 is merged.
